css/main.css

```diff
@@ -110,8 +113,40 @@ select:hover {
   display: none;
 }
 
+section#artist-writer-board {
+  column-count: 2;
+  column-gap: 0.5em;
+  column-gap: 1em;
+}
+
+article.header-message {
+  column-span: all;
+  text-align: center;
+  margin-bottom: 1em;
+}
+article.artist-writer-message {
+  padding: 1.5em;
+  background: white;
+  border-top-left-radius: 0.5em;
+  border-bottom-right-radius: 0.5em;
```

Comment:
I assume the border radius not on every corner is intentional

Reply:
yep! that's correct